pkgs/cronet_http/README.md

```diff
@@ -40,22 +38,45 @@ void main() async {
   final Client httpClient;
   if (Platform.isAndroid) {
     final engine = CronetEngine.build(
-        cacheMode: CacheMode.memory,
-        cacheMaxSize: 2 * 1024 * 1024,
-        userAgent: 'Book Agent');
+      cacheMode: CacheMode.memory,
+      cacheMaxSize: 2 * 1024 * 1024,
+       userAgent: 'Book Agent',
+    );
     httpClient = CronetClient.fromCronetEngine(engine);
   } else {
     httpClient = IOClient(HttpClient()..userAgent = 'Book Agent');
   }
 
-  final response = await client.get(Uri.https(
+  final response = await client.get(
+    Uri.https(
       'www.googleapis.com',
       '/books/v1/volumes',
-      {'q': 'HTTP', 'maxResults': '40', 'printType': 'books'}));
+      {'q': 'HTTP', 'maxResults': '40', 'printType': 'books'},
+    ),
+  );
 }
 ```
 
+### Use embedded Cronet
+
+The embedded [Cronet][] is provided as `org.chromium.net:cronet-embedded`
```

Comment:
Maybe something more explicit like:

By default, `package:cronet_http` uses the [Cronet][] library provided by [Google Play Services][].

If you want your application to work without [Google Play Services][], you can instead depend on the `org.chromium.net:cronet-embedded` package by using `dart-define` to set `cronetHttpNoPlay` is set to `true`.

For example:
...

Reply:
> By default, `package:cronet_http` uses the [Cronet][] library provided by [Google Play Services][].

The first paragraph duplicates the previous paragraphs. I'll take the second.